proto/ibc/lightclients/solomachine/v1/solomachine.proto

```diff
@@ -67,3 +64,86 @@ message TimestampedSignature {
   uint64 timestamp                   = 2;
 }
 
+// SignBytes defines the signed bytes used for signature verification.
+message SignBytes {
+  option (gogoproto.goproto_getters) = false;
+
+  uint64 sequence = 1;
```

Comment:
this wrapper results in marshaling twice, not sure if it's worth the abstraction

Reply:
the issue is we need to be able to grab the sequence for misbehavior verification. Any evidence of misbehaviour needs to be able to be unmarshaled into this otherwise I'm not sure how we could get the sequence. Any other ideas?